docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+networks:
+  xpd-network:
+    external: false
+
+services:
+  xpd-gateway:
+    image: ghcr.io/randomairborne/xpd-gateway:latest
+    restart: unless-stopped
+    networks:
+      - xpd-network
+    env_file:
+      - .env
+    environment:
+      - REDIS_URL=redis://redis:6379
```

Comment:
add a postgres url here, probably

Reply:
I can try this. lemme test locally before i update here

Reply:
can confirm that it doesn't work here.

Reply:
why not?

Reply:
doesn't read the user or the password. issue i was having before moving it to the .env
i am testing another location.